package.json

```diff
@@ -7,28 +16,16 @@
     "eslint-config-airbnb-base": "11.1.1",
     "eslint-plugin-import": "2.2.0",
     "eslint-plugin-json": "1.2.0",
+    "imports-loader": "0.7.1",
     "lintspaces-cli": "0.6.0",
     "stylelint": "7.9.0",
     "webpack": "2.6.1"
   },
-  "babel": {
-    "presets": [
-      [
-        "env",
-        {
-          "targets": {
-            "browsers": [
-              "ie >= 9"
```

Comment:
This should have been done in #7522. As far as I understand, this is only used by Babel CLI (Babel loader has its own separate config) and thus is no longer relevant now -- I could be wrong here.